developer/src/kmc-ldml/src/compiler/ldml-compiler-messages.ts

```diff
@@ -20,7 +23,8 @@ export class LdmlCompilerMessages {
   static Error_InvalidLocale = (o:{tag: string}) => m(this.ERROR_InvalidLocale, `Invalid BCP 47 locale form '${def(o.tag)}'`);
 
   static ERROR_HardwareLayerHasTooManyRows = SevError | 0x0003;
-  static Error_HardwareLayerHasTooManyRows = () => m(this.ERROR_HardwareLayerHasTooManyRows, `'hardware' layer has too many rows`);
+  static Error_HardwareLayerHasTooManyRows = (x: any) => mx(x,
+    this.ERROR_HardwareLayerHasTooManyRows, `'hardware' layer has too many rows`);
```

Comment:
@mcdurdin thoughts on this one?

Reply:
Looks workable to me. Formatting should match our revised layout for messages, and leave a trailing comma for the missing detail message that we might weant to add in future (trailing comma both suggests another parameter could be added, and makes for simpler code review).

```suggestion
  static Error_HardwareLayerHasTooManyRows = (x: any) => mx(x,
    this.ERROR_HardwareLayerHasTooManyRows, 
    `'hardware' layer has too many rows`,
  );
```